src/nvim/api/private/helpers.c

```diff
@@ -971,3 +970,19 @@ static void set_option_value_err(char *key,
     api_set_error(err, Exception, "%s", errmsg);
   }
 }
+
+void _api_set_error(Error *err, ErrorType errType, const char *format, ...)
+{
+    va_list args1;
+    va_start(args1, format);
+    va_list args2;
```

Comment:
No. I got my example from [here](http://en.cppreference.com/w/c/io/vfprintf). A `va_list` is a struct which contains a a pointer to the 'current' parameter. When an argument is read, this pointer increases. The `va_end` could potentially restore the `va_args` to the initial state, but this is [not](http://en.cppreference.com/w/cpp/utility/variadic/va_end) guaranteed by the standard.